src/QueueProducer/QueueProducer.csproj

```diff
@@ -9,6 +9,6 @@
     <Copyright>Copyright © Solace Corp 2022</Copyright>
```

Comment:
should we be changing all copyright dates to 2023? or just leave them as 2022?

Reply:
As discussed copyright statements is a bigger and separate issue.  There is likely a need to add Copyright lines to every file.